presto-main/src/test/java/com/facebook/presto/sql/gen/TestExpressionCompiler.java

```diff
@@ -1786,18 +1784,13 @@ private void assertExecute(String expression, Type expectedType, Object expected
         addCallable(new AssertExecuteTask(functionAssertions, expression, expectedType, expected));
     }
 
-    private void addCallable(Callable<Void> callable)
+    private void addCallable(Runnable runnable)
```

Comment:
Consider renaming

Reply:
I wanted to rename, but left it the same to minimize the diff.